packages/entity-detail/src/modules/entityDetail/sagas.js

```diff
@@ -387,7 +387,7 @@ export function* updateMarked({payload: {entityName, entityId, marked}}) {
 export function* reloadAfterAction({payload}) {
   const {definition} = payload
   if (definition.id !== 'delete') {
-    yield call(loadData, false)
+    yield call(loadData, true)
```

Comment:
yield call(loadData) also possible as true is default value of the reset argument